src/MessageHandler/DeleteFileHandler.php

```diff
@@ -25,31 +26,38 @@ class DeleteFileHandler implements MessageHandlerInterface
      */
     private $fileRepository;
 
+    /**
+     * Datastore Utility instance.
+     *
+     * @var Datastore
+     */
+    private $datastore;
+
     /**
      * Constructor for this Controller, to set up default services.
      *
      * @param LoggerInterface $deleteFileLogger Name hinted delete_file logger.
      * @param FileRepository  $fileRepository   The file Repository.
-     *
+     * @param Datastore       $datastore        Datastore utility instance.
      */
-    public function __construct(LoggerInterface $deleteFileLogger, FileRepository $fileRepository)
+    public function __construct(LoggerInterface $deleteFileLogger, FileRepository $fileRepository, Datastore $datastore)
     {
         $this->logger = $deleteFileLogger;
         $this->fileRepository = $fileRepository;
+        $this->datastore = $datastore;
     }
 
     public function __invoke(DeleteFile $deleteFile)
     {
-        $fileId = $deleteFile->getFileId();
+        $fileId = $deleteFile->getFilePath();
         $this->logger->info(sprintf('Processing File with ID: %d', $fileId));
```

Comment:
This won't work now, because %d is decimal, and the path is string.

Reply:
Fixed it